dspace-server-webapp/src/main/java/org/dspace/app/rest/converter/MetadataConverter.java

```diff
@@ -138,11 +139,39 @@ private <T extends DSpaceObject> void persistMetadataRest(Context context, T dso
             String element = seq[1];
             String qualifier = seq.length == 3 ? seq[2] : null;
             for (MetadataValueRest mvr: entry.getValue()) {
-                dsoService.addMetadata(context, dso, schema, element, qualifier, mvr.getLanguage(),
-                        mvr.getValue(), mvr.getAuthority(), mvr.getConfidence());
+                addMetadataByService(dsoService, context, dso, schema, element, qualifier, mvr);
             }
         }
         dsoService.update(context, dso);
     }
 
+    /**
+     * Call `addMetadata` method by arbitrary dso service. ItemService calls `addMetadata` method with more
+     * arguments than other services.
+     *
+     * @param dsoService service with implemented `addMetadata` method
+     * @param context the context to use.
+     * @param dso the DSpace object.
+     * @param schema name of the metadata schema
+     * @param element name of the metadata element
+     * @param qualifier name of the metadata qualifier
+     * @param mvr MetadataValueRest object
+     * @throws SQLException if a database error occurs.
+     */
+    private <T extends DSpaceObject> void addMetadataByService(DSpaceObjectService<T> dsoService, Context context,
+                                                               T dso, String schema, String element, String qualifier,
+                                                      MetadataValueRest mvr) throws SQLException {
+        // Use `place` argument only for ItemService because some services doesn't have implemented
+        // `addMetadata` method with `place` argument.
+        // `place` is important because of importing Items. Without it the Items metadata could be in the
```

Comment:
What about the else branch then, won't it be missing there?

Reply:
I do not understand the question. I've added the method with the same functionality, but ItemService could have additional parameter which we need.

Reply:
Thanks for in-preson explanation. We just extended vanilla version IF imported type is item. Other types do not need place.